ui-components/src/forms/FieldGroup.tsx

```diff
@@ -93,7 +93,7 @@ const FieldGroup = ({
         />
         <label
           htmlFor={item.id}
-          className={`font-semibold ${fieldLabelClassName} ${
```

Comment:
@slowbot updated! I also think it's a good idea to break the loading issue into a new ticket.

Reply:
@dominikx96 I added some additional info into the ticket about the loading issue, and I think it would be a relatively small fix in the `FormCard` component to still render the header styling even if we don't have the header yet. What do you think?

Reply:
@emilyjablonski makes sense! I notice in a couple of places there is no header so I think I need to refactor the `FormCard` props to determine whether the header should be visible and what value should be inside.